nav2_smac_planner/test/test_smac_2d.cpp

```diff
@@ -70,6 +71,20 @@ TEST(SmacTest, test_smac_2d)
   } catch (...) {
   }
 
+
+  goal.pose.position.x = 0.0;
```

Comment:
Where's the test in this? Also lots of white space that will cause the linter to be unhappy

Reply:
merge artefact, tests are in `nav2_system_tests` `test_planner_plugins`